utils/dataSet.ts

```diff
@@ -12,6 +12,7 @@ export const apiDataSet = {
     updatedStreamTitle: 'My updated title',
     isoDate: new Date().toISOString(),
     randomWord: `word${(Math.random() + 1).toString(36).substring(7)}`,
+    guestUserToken: '18be5c07fb38ddbb9515777a047273977da1e767f7d5d066917e4636f757462e7fe0e5c383df0a27c610f6283ef1a427cda15014f7f384e125b349f9b832c00a1234', //only for the Negative Flows 
```

Comment:
@YulianH guestUserToken to guestUserTokenNegativeFlow

Reply:
done